src/input_buttons.h

```diff
@@ -73,6 +73,7 @@ namespace Input {
 		FAST_FORWARD_PLUS,
 		TOGGLE_FULLSCREEN,
 		TOGGLE_ZOOM,
+		DEBUG_ABORT_EVENT,
```

Comment:
could you move this next to the other DEBUG stuff?

Reply:
Fixed.

Reply:
sorry, just noticed you must also add it to ``kButtonNames`` and ``kButtonHelp``. They will be used by the settings scene later

Reply:
Added `DEBUG_ABORT_EVENT` to `kButtonNames`and `kButtonHelp` now. While this PR doesn't add the settings menu, `SETTINGS_MENU` has been added to `kButtonNames` because `kButtonNames` and `kButtonHelp` were in a inconsistent state (39 vs. 40 entries). In `kButtonHelp` was already a description for `SETTINGS_MENU` while this one was missing in `kButtonNames`.

Reply:
yeah, I noticed something similiar with the key names when I readded the input display to the settings scene: When I added additional keyboard keys I forgot to add them to names and help, making the settings-input scene instant-crash...